client/src/index.js

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { createBrowserHistory } from 'history';
-import Routes from 'routes';
-import configureStore from 'store';
-import registerServiceWorker from 'registerServiceWorker';
+import Routes from './routes';
```

Comment:
don't need this if you pull in my changes with the .env file so that you can use absolute imports instead of relative